NOTICE

```diff
@@ -9,3 +9,159 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 
 See the Agreement for the specific language governing permissions and
 limitations under the Agreement.
+
+Below is a list of third party open source software which is provided by
+Mesosphere in connection with Mesosphere DC/OS.
+
+Below is a list of third party open source software which is provided by Mesosphere in connection with Mesosphere DC/OS.
```

Comment:
This is a repeat of the text in the previous line

Reply:
@margaret  - thanks for spotting. Fixed.